packages/flame_svg/lib/svg_component.dart

```diff
@@ -30,7 +35,7 @@ class SvgComponent extends PositionComponent {
 
   @override
   void render(Canvas canvas) {
-    _svg?.render(canvas, size);
+    _svg?.render(canvas, size, overridePaint: paint);
```

Comment:
Lets make it a non-optional parameter and remove `_paint` from `Svg` altogether

Reply:
done

Reply:
We shouldn't, `Svg` can be used outside of FCS.

Reply:
Sorry @dipakp2726, the paint needs to exist on both just like it was. 😅

Reply:
reverted.